portfolio/src/main/webapp/script.js

```diff
@@ -26,6 +26,70 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function expandDesign(clickedId){
```

Comment:
indentation is a little off on the contents of this function. The function declaration on this line is good, but everything below it should be indented by one tab.